tests/Domain/Composer/Version/DrupalCoreVersionResolverTest.php

```diff
@@ -15,19 +15,26 @@
 use Prophecy\Prophecy\ObjectProphecy;
 
 /**
- * @property \Acquia\Orca\Domain\Composer\Version\DrupalDotOrgApiClient|\Prophecy\Prophecy\ObjectProphecy $drupalDotOrgApiClient
- * @property \Acquia\Orca\Domain\Composer\Version\VersionSelectorFactory|\Prophecy\Prophecy\ObjectProphecy $selectorFactory
- * @property \Composer\Package\PackageInterface|\Prophecy\Prophecy\ObjectProphecy $package
- * @property \Composer\Package\Version\VersionSelector|\Prophecy\Prophecy\ObjectProphecy $selector
+ * @property \Acquia\Orca\Domain\Composer\Version\DrupalDotOrgApiClient|\Prophecy\Prophecy\ObjectProphecy
+ *   $drupalDotOrgApiClient
+ * @property \Acquia\Orca\Domain\Composer\Version\VersionSelectorFactory|\Prophecy\Prophecy\ObjectProphecy
+ *   $selectorFactory
+ * @property \Composer\Package\PackageInterface|\Prophecy\Prophecy\ObjectProphecy
+ *   $package
+ * @property \Composer\Package\Version\VersionSelector|\Prophecy\Prophecy\ObjectProphecy
+ *   $selector
```

Comment:
Ditto above on wrapping `@property` annotations. Also, this sort of change is meaningless, i.e., it doesn't make any difference to the PR, so it just creates noise. It should be avoided.

Reply:
Done